app/models/permissions/technician_permission.rb

```diff
@@ -5,10 +5,41 @@
 #
 module Permissions
   class TechnicianPermission < ScientistPermission
+    # - Technician's cannot edit Admin users or set users to admin type
+    # - Technician's cannot edit the protect flag within Locations
     def initialize(user)
       super
       allow :move_locations, [:create]
-      allow :empty_locations, [:create]
+      allow :location_types, [:create, :update]
+      allow :teams, [:create, :update]
+      allow :users, [:create, :update] do |record|
+        check_user(record)
+      end
+      allow :locations, [:create, :update] do |record|
+        check_locations(record)
+      end
+    end
+
+    private
+
+    def check_user(record)
+      # If user type is creating an admin or is editing an admin then error
+      if record.user.type == "Administrator" || record.user.instance_of?(Administrator)
+        record.errors.add(:base, "Technician's cannot create/edit admin users")
+        false
+      else
+        true
+      end
+    end
+
+    def check_locations(record)
```

Comment:
As above. Also it might be worth having a different allow for create or update which might keep things simpler.